.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "node": true,
+    "jest/globals": true
+  },
+  "extends": ["eslint:recommended", "plugin:jest/recommended"],
+  "overrides": [
+    {
+      "files": ["**/*.test.js", "**/*.cy.js"],
+      "env": {
+        "jest": true,
+        "node": true,
+        "cypress/globals": true
+      },
+      "plugins": ["jest", "cypress"],
+      "extends": ["plugin:cypress/recommended"],
```

Comment:
You should have two override objects, one for cypress and one for jest. Instead you have them merged together into a single object which may give unwanted effects.